### PR TITLE
polish touch feedback on iOS

### LINKED ISSUE #306: Links flash white on ios when thumbing/scrolling through app.
## Acceptance Criteria

- [ ] Upon scrolling in app, when touching different sections(links) default white "flash" should not occur.

## Context

Currently, when you thumb through the app on ios, anytime that you touch a link, the whole element flashes white.

### PR DESCRIPTION
**Pull request by conrad-vanl**
Fixes #306 by adding a very small delay in touch feedback, which prevents touch feedback from triggering as you're scrolling a feed. Also lowers the opacity affect on touch a bit so that the touch feedback isn't as drastic.

## Creator

- [ ] Build relevant tests if any apply
- [ ] Ensure there are no new warnings
- [ ] Upload an animated GIF showcasing the solution (if it applies)
- [ ] Set a relevant reviewer

## Reviewer

- [x] Run `test` and make sure all tests pass
- [x] Review function and form on iOS
- [x] Review function and form on Android
- [x] Review function and form on Web
- [x] Review new test logic

### CODE REVIEW

File: src/@ui/Touchable/index.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
 import { Platform, TouchableNativeFeedback, TouchableOpacity } from 'react-native';
+import { defaultProps } from 'recompose';
 
-const Touchable = Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+const IOSTouchable = defaultProps({
+  delayPressIn: 50,
```
Comment (burkeshartsis) [inline src/@ui/Touchable/index.js]: @conrad-vanl In stead of making this default could we just pass it in on touchables in feeds like cards?

Comment (conrad-vanl) [reply src/@ui/Touchable/index.js]: We could... granted, i'd say that there are more touchables in scrollviews then there are touchables not in scrollviews. I'd almost rather see about how low we can lower the delay value and still keep scrollviews from firing touch effects on scrolling.

Comment (burkeshartsis) [reply src/@ui/Touchable/index.js]: Yeah I looked into this a little bit to see if there might be a better way to address this given that delaying user interaction is usually the last resort approach. Unfortunately, the way react native works by "handing off" the touch event from components within the `ScrollView` to the `ScrollView` itself on drag is pretty hard to work around leaving no other options than to add a delay like you have. I might play look at playing with the `delayPressIn` value a bit if I get a chance.

**Comment by conrad-vanl** [comment]
@burkeshartsis in that case, lets just use `TouchableWithoutFeedback` for cards for now. That's where the main issue is. We can pass a `component` prop into the wrapping `<Link>`

**Pull request merged**